ui/app/styles/components/loading-spinner.scss

```diff
@@ -33,8 +33,8 @@ $darker-side: darken($lighter-side, 15%);
     align-items: center;
 
     .icon {
-      width: 32%;
-      height: 32%;
+      width: 34%;
+      height: 34%;
```

Comment:
The loading spinner: maybe you've never seen it because Nomad loads so fast!

But there is a Nomad N icon superimposed over a spinning cube and the new icon is of slightly smaller stature than the old one, and lacks the previous one's implicit transparency.